src/nim_status.nim

```diff
@@ -4,7 +4,8 @@ from nim_status/types import
 from nim_status/lib/shim as nim_shim import
   hashMessage,
   generateAlias,
-  identicon
+  identicon,
+  createAccount
```

Comment:
At this time, new pure Nim implementations should probably not be exposed at the top-level *unless* they're accompanied by tests that show the Nim replacement gives identical output for the same input compared to the Go implementation. Instead they can be imported (for usage, testing) from a nim module in `lib/`. No big change needed, just delete this. Also, if we did keep this we'd need to delete the `createAccount` imported/exported from the go shim.

Also, there has been some reorg and it was encouraged on the call earlier this week that the `feat/settings` branch be used as the base. I guess we can live with it being based off `master` since it's not a big PR.

Reply:
As an example of "gives identical output..." tests see what's done for `generateAlias`:

https://github.com/status-im/nim-status/blob/master/tests/c/shims.c#L84

https://github.com/status-im/nim-status/blob/master/tests/nim/shims.nim#L84

Reply:
One thing re: "gives identical output"... for this function that's a challenge unless both the Go and Nim implementations allow a seed to optionally be supplied to make the generation deterministic. That may not be possible — I'd need to look at the status-go API and what's possible with the Nim libs we're using. If it's not possible then, that's fine, but we'd want to come up with a test that demonstrates that Nim and Go implementations have the same shape or something like that.